wcc-contentful-app/app/views/components/_menu-item.html.erb

```diff
@@ -2,35 +2,40 @@
 
    classes ||= ['nav-item']
    classes.push('active') if item_active?(item)
-   classes.push('dropdown') if dropdown?(item) %>
+   classes.push('dropdown') if dropdown?(item)
+
+   tabindex ||= 0 %>
 <%= content_tag(:li, class: classes) do %>
   <% if dropdown?(item) %>
     <% if item.label %>
       <%= render_button(item.label,
             class: 'nav-link dropdown-toggle',
             data: { toggle: 'dropdown' },
-            role: 'button') %>
+            role: 'button',
+            tabindex: tabindex) %>
     <% else %>
       <%= content_tag(:a, item.name,
             class: 'nav-link dropdown-toggle',
             data: { toggle: 'dropdown' },
-            role: 'button') %>
+            role: 'button',
+            tabindex: tabindex) %>
     <% end %>
 
     <div class='dropdown-menu'>
       <% item.items.each do |dropdown_item|
```

Comment:
Are you not wanting to modify the tab index in these loops?

Reply:
We want all the tabindexes to be zero so that they go in page order